zktest/test/java/org/zkoss/zktest/zats/test2/F86_ZK_136Test.java

```diff
@@ -0,0 +1,61 @@
+/* F86_ZK_136Test.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Fri Aug 24 12:34:03 CST 2018, Created by charlesqiu
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+public class F86_ZK_136Test extends WebDriverTestCase {
+
+	@Test
+	public void test() {
+		connect();
+		JQuery toggleIndeterminate = findButtonByContent("toggleIndeterminate");
+		JQuery isIndeterminate = findButtonByContent("isIndeterminate");
+		JQuery isChecked = findButtonByContent("isChecked");
+
+		click(toggleIndeterminate);
+		waitResponse();
+		clickAndVerifyLog(isIndeterminate, "true");
+		clickAndVerifyLog(isChecked, "false");
+
+		click(jq(".z-checkbox"));
+		waitResponse();
+		clickAndVerifyLog(isIndeterminate, "false");
+		clickAndVerifyLog(isChecked, "true");
+
+		click(toggleIndeterminate);
+		waitResponse();
+		clickAndVerifyLog(isIndeterminate, "true");
```

Comment:
toggle `Indeterminate` should not affect `checked`

https://www.w3.org/TR/2014/WD-html51-20140617/forms.html#checkbox-state-(type=checkbox)
> The control is never a true tri-state control, even if the element's indeterminate IDL attribute is set to true. The indeterminate IDL attribute only gives the appearance of a third state.